db/migrate/20200912190210_update_weight_calculators.rb

```diff
@@ -1,10 +1,12 @@
 class UpdateWeightCalculators < ActiveRecord::Migration
   def change
+    Spree::Calculator.connection.execute(
+      "UPDATE spree_preferences SET key = replace( key, 'per_kg', 'per_unit') WHERE key like '/calculator/weight/per_kg/%'"
```

Comment:
I'd rather user `ilike` because skipping case-sensitiveness is slightly faster. That's why I make it my default when writing SQL statements. But I bet it won't make much difference in this case, so up to you.

Reply:
Might as well get in the good habit :)